compiler/rustc_lint/src/internal.rs

```diff
@@ -108,7 +108,7 @@ impl<'tcx> LateLintPass<'tcx> for TyTyKind {
                                     lint.build(&format!("usage of qualified `ty::{}`", t))
                                         .span_suggestion(
                                             path.span,
-                                            "try using it unqualified",
+                                            "you must import it and use it unqualified",
```

Comment:
```suggestion
                                            "consider importing it and using it unqualified",
```
must feels a bit strong to me :laughing: one could disable the lint as an alternative

r=me after that

Reply:
I went with the original "try" instead of "consider", otherwise: :+1: